tests/test_bgp_af.rb

```diff
@@ -893,13 +614,21 @@ def test_network
     vrfs.each do |vrf|
       afs.each do |af|
         dbg = sprintf('[VRF %s AF %s]', vrf, af.join('/'))
+        config_ios_xr_dependencies(1)
         af_obj = RouterBgpAF.new(1, vrf, af)
         network_cmd(af_obj, dbg)
+
+        af_obj.destroy
       end
     end
   end
 
   def network_cmd(af, dbg)
+    if platform == :ios_xr
+      %w(rtmap1 rtmap2 rtmap3 rtmap5 rtmap6 rtmap7).each do |policy|
+        config("route-policy #{policy}", 'end')
+      end
+    end
     # Initial 'should' state
     if /ipv6/.match(dbg)
```

Comment:
I know this is base code but since you're rewriting big chunks anyway - why are we regexping against the dbg string instead of checking `af.afi == 'ipv6'`?